docs/src/examples/ValidatedFormExample/ValidatedFormExample.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { ValidatedForm, FormLayout, ValidatedTextField, Button, ButtonGroup } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+
+export interface IProps{
+  form: {
+    getFieldProps: any,
+    getFieldError: any,
+    validateFieldsAndScroll: any,
+  };
+  style?: React.CSSProperties;
+  onSubmit: (values: [any]) => void;
+  onSubmitError: (values: [any], error: Error) => void;
+}
+export interface IState {
+  appName: string;
+  appDescription: string;
+}
+
+class ValidatedFormExample extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+
+    this.state = {
+      appName: '',
+      appDescription: '',
+    };
+  }
+
+  valueUpdater(field: any) {
+    return (value: any) => this.setState({ [field]: value });
+  }
+
+  render() {
+    return (
+      <div className={styles.example}>
+        <ValidatedForm>
+          <FormLayout>
+            <ValidatedTextField
+              id="AppName"
+              required={true}
+              label="App Name"
+              placeholder=""
+              helpText="We recommend keeping your app name under 23 characters."
+              onChange={this.valueUpdater('appName')}
+              value={this.state.appName}
+              name="App Name"
+              validateTrigger={['onBlur']}
+              validateRules={[
+                { required: true, message: 'App Name is required.' },
+              ]}
+            />
+            <ValidatedTextField
+              multiline
+              id="appDescription"
+              required={true}
+              name="App Description"
+              value={this.state.appDescription}
+              label="App Description"
+              placeholder=""
```

Comment:
No need to put this here if it's going to be a blank string. Otherwise give it a value.

Reply:
I removed blank string props placeholder.